packages/app-store/_components/OmniInstallAppButton.tsx

```diff
@@ -17,24 +16,21 @@ export default function OmniInstallAppButton({
   className,
   returnTo,
   teamId,
+  onAppInstallSuccess,
 }: {
   appId: string;
   className: string;
+  onAppInstallSuccess: () => void;
   returnTo?: string;
   teamId?: number;
 }) {
   const { t } = useLocale();
   const { data: app } = useApp(appId);
-  const utils = trpc.useUtils();
 
   const mutation = useAddAppMutation(null, {
     returnTo,
     onSuccess: (data) => {
-      utils.viewer.apps.appById.invalidate({ appId });
-      utils.viewer.apps.integrations.invalidate({
-        extendsFeature: "EventType",
-        ...(teamId && { teamId }),
-      });
```

Comment:
Moved this logic to `packages/features/eventtypes/components/tabs/apps/EventAppsTab.tsx`